jams/jamongo.py

```diff
@@ -0,0 +1,183 @@
+"""
+JAMS Mongo API
+==============
+
+This library provides an interface for interfacing with a JAMS Mongo
+instance.
+
+"""
+
+import pymongo
+
+
+def convert_annotation_list(annotations_array, audio_id):
+    """Take a "Jams-document" style annotations array,
+    and convert it into a mongo-able annotationsl list.
+
+    Parameters
+    ----------
+    annotations_list : jams.AnnotationArray
+
+    audio_id : ObjectId
+        Pointer to an audio document.
+
+    Returns
+    -------
+    converted_annotations : list of Annotation
```

Comment:
Yes. We'll just have to do a lil' old `[x.__json__ for x in annotations]` before we actually insert it, but I think this is what we want.